components/02-elements/mega-menu/mega-menu.hbs

```diff
@@ -0,0 +1,55 @@
+<div class="mega-menu">
+    <ul class="mega-menu__wrapper container">
+        {{#each list}}
+            <li class="mega-menu__item {{ class }}">
+                <a href="#" class="mega-menu__link">
+                    {{text}}
+                </a>
+
+                {{#if nestedList}}
+                    <ul class="
+                            mega-menu__dropdown
+                            mega-menu__dropdown--level1
+                        "
+                    >
+                        {{#each nestedList}}
+                            <li class="
+                                    mega-menu__dropdown-item
+                                    mega-menu__dropdown-item--level1
+                                    {{#if modifier}}
+                                        {{modifier}}
+                                    {{/if}}
+                                "
+                            >
+                                <a href="#"
```

Comment:
same as above - IMO this `href` value should be set in config

Reply:
changed.